### DISCUSSION TITLE
[v5] 鼠标 hover 到标签上时，无法显示 tooltip

### DISCUSSION THREAD

**Discussion by visiky** [antvis/G2#5005] [Q&A] [answered]
## 期望结果

这个问题在 4.0 中也存在，期望鼠标 hover 在标签上时，也可以如右图展示 tooltip

|现状|期望|
|---|---|
|<img width="663" alt="image" src="https://user-images.githubusercontent.com/15646325/224605371-7d73baec-2dbc-43e2-b627-c8acf600a5ff.png">|<img width="646" alt="image" src="https://user-images.githubusercontent.com/15646325/224605402-49633c20-9701-4156-92b6-2d870a3d4d37.png">|

## 复现地址

- https://g2-site-pre.alipay.com/zh/examples/general/interval/#bar-stacked-diverging-rounded

**Comment by hustcc** [comment, marked as answer]
可以将 label 的事件禁用，不响应事件。

```ts
.label({
  text: 'value',
  position: 'inside',
  formatter: (v) => (v ? `${v}%` : ''),
  transform: [{ type: 'overlapDodgeY' }],
  style: {
    fill: '#000',
    fontSize: 10,
    pointerEvents: 'none',  // 👈🏻，不响应事件即可
  },
});
```
